.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.7
+          python-version: "3.10"
```

Comment:
Do we also need to update the devcontainer?

Reply:
Just checked – default is already 3.10 there.